.github/workflows/deploy-pr-to-s3.yml

```diff
@@ -0,0 +1,47 @@
+name: Deploy the pull request to AWS S3
+on: [pull_request]
+
+env:
+  PR_NUMBER: ${{ github.event.pull_request.number }}
+  REPO_NAME: ${{ github.event.repository.name }}
+
+jobs:
+  deploy-to-s3:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          persist-credentials: false
+
+      - name: Read .nvmrc
+        run: echo "::set-output name=NODE_VERSION::$(cat .nvmrc)"
+        id: nvm
+
+      - uses: actions/setup-node@v1
+        with:
+          node-version: ${{ steps.nvm.outputs.NODE_VERSION }}
+
+      - name: Install dependencies
+        run: npm ci
+
+      - name: Build
+        run: npm run build:storybook -- -o ./dist/${{ env.REPO_NAME }}/${{ env.PR_NUMBER }}
```

Comment:
Here I'm using the output param to pass the path where I want the build.
You can check the [docs](https://storybook.js.org/docs/react/workflows/publish-storybook#build-storybook-as-a-static-web-application) form more details.
I used it because they might change something internally to insure the app work in that sub path, but I honestly doubt that, I think a normal build could have done the job for us, but just to keep it sane, I used what they recommend.